spec/lib/addressfinder/cleanse_spec.rb

```diff
@@ -98,7 +98,7 @@
     context 'with a question mark value' do
       let(:q){ '?' }
 
-      it { expect(encoded_params).to eq('q=%3F&format=json&key=XXX&secret=YYY') }
+      it { expect(encoded_params).to eq('q=?&format=json&key=XXX&secret=YYY') }
```

Comment:
All the other API modules are happy with `URI.encode`. The alternative is to change all the others to use `ERB::Utils.url_encode` and then add an erb dependency. Why is it important that ? is encoded. I would have thought URI.encode would do as much as we needed.